config/core/kustomization.yaml

```diff
@@ -0,0 +1,90 @@
+# The core kustomization bundles up all the bits that are capi-specific. 
```

Comment:
This and everything else in the core dir is essentially what used to be in default (at least, the bits that were capi-only. not kubeadm).